src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drowable {
```

Comment:
```suggestion
public abstract class Figure implements Drowable, AreaCalculator {
```

create interface AreaCalculator with `double getArea()`

Reply:
fixed